lib/hammer_cli_foreman/auth_source_external.rb

```diff
@@ -0,0 +1,36 @@
+module HammerCLIForeman
+  class AuthSourceExternal < HammerCLIForeman::Command
+    resource :auth_source_externals
+    command_name 'external'
+    desc _('Manage external auth sources')
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _('Id')
+        field :name, _('Name')
+      end
+
+      build_options
+    end
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      output ListCommand.output_definition do
+        HammerCLIForeman::References.taxonomies(self)
+      end
+
+      build_options
+    end
+
+    class UpdateCommand < HammerCLIForeman::UpdateCommand
+      desc _('Update organization and location for Auth Source')
+      resource :auth_source_externals
+
+      success_message _('Taxonomy updated.')
```

Comment:
Users can also change the name of the auth source here, therefore, changing the message to `Successfully updated the external auth source`. Sounds better?